cvat-core/src/session.js

```diff
@@ -1452,6 +1453,16 @@
                             updatedFields = fields;
                         },
                     },
+                    dimension: {
+                    /**
+                     * @name enabled
+                     * @type {string}
```

Comment:
probably should be module:API.cvat.enums.TaskDimension

Reply:
Made changes accordingly.